backend/project/utils.py

```diff
@@ -0,0 +1,18 @@
+import requests
```

Comment:
We use basically use APIView and other related classes provided by the Django Rest framework to make APIs. For reference, you can see `backend/apps/core/views.py`